openedx/core/djangoapps/course_groups/management/commands/remove_users_from_multiple_cohorts.py

```diff
@@ -1,56 +0,0 @@
-"""
```

Comment:
This file and test_remove_users_from_multiple_cohorts.py date from over 6 months ago, when this problem was first noticed and fixed, but both are made obsolete by the changes I've made recently.